test/y18n-test.js

```diff
@@ -30,6 +30,23 @@ describe('y18n', function () {
   })
 
   describe('__', function () {
+    it('can be used as a tag for template literals', function () {
+      var __ = y18n({ // eslint-disable-line
+        directory: path.join(__dirname, 'locales')
+      }).__
+
+      try {
+        // We need to do this wrapper in case y18n gets used in runtimes
+        // without template literal support.
+        eval('__`Hello ${"Ben"} ${"Coe"}`').should.equal('Hello Ben Coe') // eslint-disable-line
```

Comment:
might be good to test a translating to another language, so we confirm that the `%s` replacement works appropriately.